gapic/src/main/com/google/gapid/image/Image.java

```diff
@@ -66,57 +86,28 @@ public int getDepth() {
     }
 
     @Override
-    public ImageBuffer getData() {
-      return ImageBuffer.EMPTY_BUFFER;
-    }
-  };
-
-  /**
-   * Contains the bytes of the {@link Image Image's} pixel data.
-   */
-  public static interface ImageBuffer {
-    public static final ImageBuffer EMPTY_BUFFER = new ImageBuffer() {
-      @Override
-      public void uploadToTexture(Texture texture) {
-        texture.loadData(0, 0, GL11.GL_RGB, GL11.GL_RGB, GL11.GL_UNSIGNED_BYTE, null);
-      }
-
-      @Override
-      public ImageData getImageData() {
-        return EMPTY_IMAGE;
-      }
+    public Image getSlice(int z) { return this; }
```

Comment:
Please add newlines after { and before }

Reply:
Done